pkg/report/linux.go

```diff
@@ -2027,40 +1992,34 @@ var linuxOopses = append([]*oops{
 			compile(`(?i)warning: .* uses (deprecated v2 capabilities|wireless extensions)`),
 			compile(`XFS \(\w+\): WARNING`),
 		},
-		crash.Warning,
 	},
 	{
 		[]byte("INFO:"),
 		[]oopsFormat{
 			{
-				title:      compile("INFO: possible circular locking dependency detected"),
-				report:     compile("INFO: possible circular locking dependency detected \\](?:.*\\n)+?.*is trying to acquire lock(?:.*\\n)+?.*at: {{PC}} +{{FUNC}}"),
-				fmt:        "possible deadlock in %[1]v",
-				reportType: crash.LockdepBug,
+				title:  compile("INFO: possible circular locking dependency detected"),
+				report: compile("INFO: possible circular locking dependency detected \\](?:.*\\n)+?.*is trying to acquire lock(?:.*\\n)+?.*at: {{PC}} +{{FUNC}}"),
+				fmt:    "possible deadlock in %[1]v",
 			},
 			{
-				title:      compile("INFO: possible irq lock inversion dependency detected"),
-				report:     compile("INFO: possible irq lock inversion dependency detected \\](?:.*\\n)+?.*just changed the state of lock(?:.*\\n)+?.*at: {{PC}} +{{FUNC}}"),
-				fmt:        "possible deadlock in %[1]v",
-				reportType: crash.LockdepBug,
+				title:  compile("INFO: possible irq lock inversion dependency detected"),
+				report: compile("INFO: possible irq lock inversion dependency detected \\](?:.*\\n)+?.*just changed the state of lock(?:.*\\n)+?.*at: {{PC}} +{{FUNC}}"),
+				fmt:    "possible deadlock in %[1]v",
 			},
 			{
-				title:      compile("INFO: SOFTIRQ-safe -> SOFTIRQ-unsafe lock order detected"),
-				report:     compile("INFO: SOFTIRQ-safe -> SOFTIRQ-unsafe lock order detected \\](?:.*\\n)+?.*is trying to acquire(?:.*\\n)+?.*at: {{PC}} +{{FUNC}}"),
-				fmt:        "possible deadlock in %[1]v",
-				reportType: crash.LockdepBug,
```

Comment:
Types were intentionally attached to patterns, so that they are always up-to-date.
How can be we keep the new scheme up-to-date?

Reply:
I am looking at the test file this PR updates, and they don't have type connection.
Let's add type to every test then (in a separate preceding change). It would be interesting to see if this change will affect any of them.

Reply:
I don't fully understand the meaning of "Let's add type to every test then".
You can see crash.KFENCE added to prove that it works. Hope it is enough to apply this PR.